dbt/profiles.yml

```diff
@@ -0,0 +1,17 @@
+pudl_dbt:
+  outputs:
+    # Define targets for nightly builds, and local ETL full/fast
+    # See models/schema.yml for further configuration
+    nightly:
+      type: duckdb
+      path: "{{ env_var('PUDL_OUTPUT') }}/pudl.duckdb"
```

Comment:
Putting a file named `pudl.duckdb` in the `PUDL_OUTPUT` directory alongside `pudl.sqlite` implies that this is the PUDL database, but in DuckDB format, which isn't the case and will be confusing -- this DB will only end up holding the models, seeds, and views we define, right?